blast_runner.py

```diff
@@ -37,7 +38,8 @@ def run_blast(query, db, out):
                      '-dbtype', "'nucl'", '-out', 'blast_{}'.format(db),
                      '-parse_seqids'])
     subprocess.call(['blastn', '-db', 'blast_{}'.format(db), 'query',
-                     '{}'.format(query), '-out', '{}'.format(out), '-outfmt', '7'])
+                     '{}'.format(query), '-out', '{}'.format(out), '-outfmt', '7']) 
```

Comment:
since `query`, and `out` are already strings, you don't have to do `'{}'.format(query)`, that's identical to just `query`